src/main/java/com/nfl/glitr/util/QueryComplexityCalculator.java

```diff
@@ -358,6 +386,28 @@ private boolean isLeaf(Node node) {
         return node.getChildren().isEmpty();
     }
 
+    public boolean isMutation(String query){
+        return query.contains(MUTATION_DEFINITION);
```

Comment:
This appears insufficient, what if there is string with the value "mutation" somewhere in the query?

Reply:
So we have two other choices.

1) check if the initial string contains 'mutation'
2) Create a document and verify through there.

Reply:
yeah, you should just check that it starts with "mutation" (ignoring whitespace). make sure to adds tests around this though.

Reply:
cool thanks